### PR TITLE
Week 6 - Weather app, blue level - Nadia Mazzoni

### PR DESCRIPTION
**Pull request by Bluepoisson**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Weather 2GO</title>
```
Comment (bloeli) [inline index.html:6]: Nice name!

File: README.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Create a weather app containing current weather for wanted city and a 5 days forecast.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Learning how to fetch an api and displaying wanted data was quite simple until I encountered an array and had to format time and dates. My knowledge was base being far too slim combined with my stubborness made me loose valuable time.
```
Comment (bloeli) [inline README.md:7]: Agree, I googled some and found a solution finally

**Review by bloeli — CHANGES_REQUESTED** [review]
Nice that you are stubborn and make progress even though I think you have felt a bit frustrated, at least I do when I wan't to understand and make it on my own/are stubborn and things maybe don't go as fast and doesn't work the way I would like it. But I guess not giving up, and ask for help (kastrull as you said today haha) is important and I think you have a great spirit beeing stubborn and also not giving up, but don't forget about the kastrull :) I feel a lot frustrated understanding Javascript, even though I made a weather app that works kind of well, I can feel that I now does'nt really know what I've done everywhere. I google a lot, ask for help sometimes, and also check other peoples code, try implement what I found, understand it and also test/play around. Then I aim for doing some practice and learn Javascript at codeacademy and w3schools, and hope it will make more sense a long the way. Maybe this wasn't the greatest code review since I struggle my self with Javascript, and cannot give you tips on how to solve things, but I hope and belive you will solve this, asking for help,, looking at other peoples code/projects and help from code coachers :) <3

File: index.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <link rel="stylesheet" href="style.css" />
```
Comment (EmOlsz) [inline index.html:7]: Let's not forget about `./` in path to file

File: script.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    hour12: false
+  });
+}
+
+// FETCHING WEATHER DATA FROM API PROVIDER : CURRENT +  FORECAST
+fetch(API_CURRENT)
+  .then((response) => {
+    return response.json();
+
+
```
Comment (EmOlsz) [inline script.js:34]: Let's delete those empty lines

File: script.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  })
+  .then(weatherToday => {
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply script.js:36]: Nope, since it's one argument we don't need to wrap it in extra parentheses

File: script.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+  });
+
+
+
+//Forecast variable :
+const fiveDays = document.getElementsByClassName("five-day")
+
+
+
```
Comment (EmOlsz) [inline script.js:53]: Once again, too much empty lines. One is completely enough

File: script.js [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+
+
+
+
+
+
```
Comment (EmOlsz) [inline script.js:98]: Aaand over here ;)

File: style.css [hunk 8]
```diff
@@ -0,0 +1,33 @@
+* {
+  font-family: monospace, sans-serif;
+}
+
+#container {
```
Comment (EmOlsz) [inline style.css:5]: We should avoid styling by id's and stick only to classes

File: style.css [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+  width: 80%;
+  height: 500px;
+  margin-left: 20px;
+  padding-left: 70px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  color: rgb(300, 220, 200);
+  background-image: url(./images/parasol.png);
```
Comment (EmOlsz) [inline style.css:14]: It's more safe to wrap url in quotes, some browsers might have problem with understanding path without them :

```
background-image: url("./images/parasol.png");
```

File: style.css [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+  background-size: cover;
+  font-weight: bolder;
+  font-size: 20px;
+}
+
+#five-day {
+  margin-top: -300px;
```
Comment (EmOlsz) [inline style.css:21]: It's almost always possible to use different approach than negative margins - maybe opposite margin on different element above?

**Review by EmOlsz — APPROVED** [review]
Good job in general Nadia :) You've met basic requirements.
Please, take a look on my comments, hope they will help you improve!

**Pull request closed without merging**